docs/user_manual/orders.md

```diff
@@ -72,8 +71,8 @@ Click "Save".
 
 There are two ways to create orders in bulk:
 
-* From the "Tracking" list, select "Pools" and check the boxes beside the pools
-  needing orders and click "Create Orders".
+* From the "Tracking" list, select "Pools". On the pools list, check the boxes
```

Comment:
Remove `From the "Tracking" list, select "Pools".` This could also be done from the Pools list on the Edit Project page. Capital on "Pools list" also